src/entities/activity/ui/items/ActionPlan/phrasalData.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Was this meant to be committed? I don't encounter any new prettier errors or warnings with this line removed:
```suggestion
```